examples_lib/encryption.go

```diff
@@ -9,10 +9,25 @@ import (
 // The 256-bit encryption key, stored as a string for convenience.
 const encryption_key = "0123456789abcdeF0123456789abcdeF"
 
-func createEncryptedArray(dir string) {
+func getContextConfiguredForEncryption() *tiledb.Context {
```

Comment:
Since we now have a function to create a config from a dict, we could use that here.
Also maybe "build" instead of "get" for a more descriptive verb?

Reply:
Thanks, replaced with `NewContextFromMap`